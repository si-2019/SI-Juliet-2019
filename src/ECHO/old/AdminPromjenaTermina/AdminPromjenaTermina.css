.AdminPromjenaTermina {
    text-align: center;
  }
  .App-header {
    background-color: #282c34;
    min-height: 15vh;
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    font-size: 10px;
    color: white;
  }
  #nazad {
    border-radius: 8px;
    background-color:  #282c34;
    font-size: 8px;
    color: white;
    width : 12vw;
    left : 15vw;
    top : 75vh;
    position: absolute;
    border: 1px solid black; 
  }
    #pon, #uto{
    background-color:  #282c34;
    border-radius: 1px;
    margin-left: 40vw;
    margin-top: 10vh;
    width : 20vw;
    min-height: 5vh;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    font-size: 8px;
    color: white;
    border : 1px solid, black;
  }
  #sviprofesori, #terminijednogprofesora{
    width : 15vw;
    min-height: 5vh;
    align-items: center;
    justify-content: center;
    font-size: 12px;
    border : 1px solid, black;
  }
  #sviprofesori{
    left: 43vw;
    top: 34vh; 
    position : absolute;
  }
  #terminijednogprofesora{
    left: 43vw;
    top: 50vh; 
    position : absolute;
  }
  .App-edit {
    border: 1px solid black; 
    background-color: darkred;
    border-radius: 2px;
    min-height: 2vh;
    left: 39vw;
    width: 10vw;
    flex-direction: column;
    align-items: center;
    top: 55vh;
    position : absolute;
    justify-content: center;
    font-size: 7px;
    color: white;
  }
