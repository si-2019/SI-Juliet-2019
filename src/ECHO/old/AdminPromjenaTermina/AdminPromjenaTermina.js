import React, { Component } from 'react';
import './AdminPromjenaTermina.css';
class AdminPromjenaTermina extends Component {
  render() {
    return (
      <div className="App">
        <header className="App-header">
          <h1> Odabir osoblja i promjena termina nastave.</h1>
        </header>
        <div id = "pon"><h1>Odaberite člana osoblja</h1></div>
        <select id = "sviprofesori">
         <option value="Novica Nosovic">Novica Nosovic</option>
        </select>
        <div id = "uto"><h1>Odaberite termin za brisanje</h1></div>
        <select id = "terminijednogprofesora">
         <option value="Srijeda, 18:00">Srijeda, 18:00</option>
        </select>
        <button className="App-edit" id = "b1"><h2>Obrišite termin</h2></button>
        <button id = "nazad"><h1>Nazad</h1></button>
      </div>
    );
  }
}
export default AdminPromjenaTermina;