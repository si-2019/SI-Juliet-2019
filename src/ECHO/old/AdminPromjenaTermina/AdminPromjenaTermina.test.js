import React from 'react';
import AdminPromjenaTermina from './AdminPromjenaTermina';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';
Enzyme.configure({ adapter: new Adapter() });
it("prikazuje formu za promjenu termina", () => {
    const component = shallow(<AdminPromjenaTermina />);
    expect(component).toMatchSnapshot();
  });
it("postoji li button za brisanje termina", () => {
    const wrapper = shallow(<AdminPromjenaTermina />);
    expect(wrapper.find("#b1").exists());
  });
it("postoji li select za odabir osoblja", () => {
    const wrapper = shallow(<AdminPromjenaTermina />);
    expect(wrapper.find("#sviprofesori").exists());
  });
it("postoji li button za nazad", () => {
    const wrapper = shallow(<AdminPromjenaTermina />);
    expect(wrapper.find("#nazad").exists()).toBe(true);
  });