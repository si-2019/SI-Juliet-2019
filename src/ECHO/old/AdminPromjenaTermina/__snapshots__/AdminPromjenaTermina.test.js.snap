// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prikazuje formu za promjenu termina 1`] = `ShallowWrapper {}`;
